### PR TITLE
Org page to allow editing of additional fields and owners

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Review by techsmyth — APPROVED** [review]
Working nicely. It does now send up the nameID as part of the update request, which actually showed up a bug on the server - fixed in https://github.com/alkem-io/server/pull/1362.

File: src/pages/Admin/Organisation/OrganisationOwnerAuthorizationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      >
+        {(entities, actions, state) => (
+          <EditMembers
+            members={entities.allMembers}
+            availableMembers={entities.availableMembers}
+            addingMember={state.addingOwner}
+            removingMember={state.removingOwner}
+            executor={entities.currentMember}
+            onAdd={actions.handleAssignAdmin}
+            onRemove={actions.handleRemoveAdmin}
```
Comment (atstojanov) [inline src/pages/Admin/Organisation/OrganisationOwnerAuthorizationPage.tsx:46]: Should be `handleAssignOwner` and `handleRemoveOwner`

File: src/pages/Admin/Organisation/OrganisationOwnerAuthorizationPage.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import EditMembers from '../../../components/Admin/Community/EditMembers';
+import OrganizationMembers from '../../../containers/organisation/OrganizationMembers';
+import { useUpdateNavigation } from '../../../hooks';
+import { AuthorizationCredential } from '../../../models/graphql-schema';
+import { OrganizationRouteParams } from '../../../routing/admin/organisation/organization';
+import OrganisationAuthorizationPageProps from './OrganisationAuthorizationPageProps';
+
+export const OrganisationOwnerAuthorizationPage: FC<OrganisationAuthorizationPageProps> = ({ paths }) => {
```
Comment (atstojanov) [inline src/pages/Admin/Organisation/OrganisationOwnerAuthorizationPage.tsx:12]: UK->US

File: src/routing/admin/organisation/OrganisationAuthorizationRoute.tsx [hunk 3]
```diff
@@ -12,8 +13,11 @@ const OrganisationAuthorizationRoute: FC<OrganisationAuthorizationRouteProps> =
 
   return (
     <Switch>
-      <Route exact path={`${path}/:role`}>
-        <OrganisationAuthorizationPage paths={currentPaths} />
+      <Route exact path={`${path}/admins/:role`}>
+        <OrganisationAdminAuthorizationPage paths={currentPaths} />
+      </Route>
+      <Route exact path={`${path}/owners/:role`}>
+        <OrganisationOwnerAuthorizationPage paths={currentPaths} />
```
Comment (atstojanov) [inline src/routing/admin/organisation/OrganisationAuthorizationRoute.tsx:20]: the `:role` parameter is redundant. The previous part of the path contains the actual role and the parameter is never used.

**Review by atstojanov — CHANGES_REQUESTED** [review]

**Review by atstojanov — APPROVED** [review]

**Pull request merged**